pkg/aws/policies.go

```diff
@@ -399,6 +399,10 @@ func (c *awsClient) createPolicy(policyArn string, document string, tagList map[
 }
 
 func (c *awsClient) IsPolicyCompatible(policyArn string, version string) (bool, error) {
```

Comment:
Can we add a basic test for this func before we merge.

Reply:
Yeah, just to that func we can do. The flow itself would require a lot of additions